react/features/authentication/components/web/LoginDialog.tsx

```diff
@@ -102,8 +96,7 @@ class LoginDialog extends Component<IProps, IState> {
 
         this.state = {
             username: '',
-            password: '',
-            loginStarted: false
```

Comment:
Why did we remove the `loginStarted` flag?

Reply:
Cause we do not need it, it is handled by the _connecting prop.